src/content/enterprise/aks.mdx

```diff
@@ -232,7 +232,7 @@ The output will look something like this:
 
 ```
 NAME                                  TYPE           CLUSTER-IP      EXTERNAL-IP                           PORT(S)          AGE
-okteto-okteto-enterprise-buildkit     LoadBalancer   10.245.142.73   a519c8b3b27f95...aks.microsoft.com    1234:32449/TCP   5m
+okteto-okteto-buildkit     LoadBalancer   10.245.142.73   a519c8b3b27f95...aks.microsoft.com    1234:32449/TCP   5m
```

Comment:
Would it be possible to change this to: `okteto-buildkit`?

Reply:
It's possible, but I think it makes the installation a bit harder.
The name is composed by `release name + chart name + component name`. Since we use `okteto` as the example release name, we'd need to change that along the tutorial. 

Do you think there's a better example name we can come up with?

Reply:
@AgustinRamiroDiaz this requires a product change, but many charts remove the duplicated release_name-chart_name from the resources if it's the same. The logic for this comes on the default helm chart (or at least it used to when I wrote the initial chart). It does look odd to have okteto-okteto in all the resources.

Reply:
This is not easy lift the RC is cut and tested and ready to go out the door. We've already scheduled downtime for the affect a rename has on cloud and I'd prefer not to delay that as it involves customer comms. We've always had it start with `okteto-okteto...` as well so given that it would affect timeline I wouldn't do this without a large reason to do so.

Reply:
Sounds good @jmacelroy . Can we update the sample to remove the stutter? We can name the release something else on the samples.

Reply:
@AgustinRamiroDiaz this is actually just a docs bug based on my testing. The code at https://github.com/okteto/app/blob/master/chart/okteto-enterprise/templates/_helpers.tpl#L19 actually removes the stutter if the chart name and release name are the same and when I install the latest 1.0 RC and name it `okteto` the buildkit will be `okteto-buildkit-.....`

In fact if you install the 0.15 chart and name the release `okteto-enterprise` it also fixes the stutter :)

Reply:
This is great news!

Reply:
Thanks @jmacelroy for checking that! I'll update the pr with the correct logs
Thanks also to @RinkiyaKeDad for noticing